tests/indices/base/test_base.py

```diff
@@ -0,0 +1,115 @@
+"""Test keyword table index."""

```

Comment:
tests are pretty comprehensive! 

why are tests under base/test_base.py? i'd just put file in `tests/indices/test_save_load.py`

Reply:
I'll move it rn